src/IO/H5/Python/CombineH5.py

```diff
@@ -36,15 +36,34 @@
     ),
     help="combined output filename (omit file extension)",
 )
+@click.option(
+    "--check_src",
+    "-c",
+    required=True,
+    type=bool,
+    help=(
+        "flag to check src files, True implies src files exist and can be"
+        " checked, False implies no src files to check."
+    ),
+)
+def combine_h5_command(file_prefix, subfile_name, output, check_src):
+    """Combines multiple HDF5 volume files"""
 
-def combine_h5_command(file_prefix, subfile_name, output):
     """This executable is used for combining a series of HDF5 volume files into one
     continuous dataset to be stored in a single HDF5 volume file."""
 
+    # Print available subfile names and exit
+    filename = file_prefix + "0.h5"
+    if not subfile_name:
+        import rich.columns
+
+        rich.print(rich.columns.Columns(filename.all_vol_files()))
```

Comment:
This will produce an error because `filename` is a string. You have to open the H5 file first. Please add an invocation to the `test_cli` test that omits the subfile name so this gets tested.

Reply:
Could you explain what you mean by omitting the subfile_name? Also, if the filename string does not work, and i am currently only inputting file_prefix, how should I add functionality?

Reply:
Where should the subfile names get printed? I did add another test which does not include the subfile name but it doesn't;t print anything at all

Reply:
You can check `result.output`. That's where the output is captured.